### PR TITLE
docs: user should restart docusaurus after adding prism additionalLanguage

### PR DESCRIPTION
**Pull request by tohidnateghi**
if you add php language and don't restart your server, you will get an error like this

uncaught Error: Cannot find module './prism-php'

until you restart your server

## Motivation

(Write your motivation here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/docs/guides/markdown-features/markdown-features-code-blocks.mdx [hunk 1]
```diff
@@ -81,6 +81,8 @@ module.exports = {
 };
 ```
 
+Aftre adding `additionalLanguages` you may restart your server if you are running your project in a local server.
```
Comment (slorber) [inline website/docs/guides/markdown-features/markdown-features-code-blocks.mdx:84]:
```suggestion
After adding `additionalLanguages`, restart Docusaurus.
```

**Comment by slorber** [comment]
thanks 👍

File: website/docs/guides/markdown-features/markdown-features-code-blocks.mdx [hunk 2]
```diff
@@ -81,7 +81,7 @@ module.exports = {
 };
 ```
 
-Aftre adding `additionalLanguages` you may restart your server if you are running your project in a local server.
+After adding `additionalLanguages`, restart Docusaurus.
```
Comment (tohidnateghi) [inline website/docs/guides/markdown-features/markdown-features-code-blocks.mdx:84]: thank you :)

**Review by tohidnateghi — COMMENTED** [review]
thank you

**Pull request merged**